VisualStudio.gitignore

```diff
@@ -0,0 +1,428 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Why are we changing the .gitignore settings? This is already sorted out for this project right?